.gitignore

```diff
@@ -1,6 +1,7 @@
 # Compiled Object files, Static and Dynamic libs (Shared Objects)
 *.o
 *.a
+/.idea
```

Comment:
Remove from the project gitignore. Things like this are usually better suited in your global ignore file.